src/main/java/org/petrax/models/PetProfile.java

```diff
@@ -0,0 +1,193 @@
+package org.petrax.models;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import java.util.Date;
+import java.util.Objects;
+
+@Entity
+public class PetProfile {
+
+    //private fields petId, name, breed, age, weight, birthday, medications, allergies, microchip number, and diagnoses
+    @Id
+    @GeneratedValue
+    private int petId;
+
+    @NotBlank(message = "Name is required")
+    @Size(min = 3, max = 50, message = "Name must be between 3 and 50 characters")
+    private String name;
+
+    @NotBlank(message = "Breed is required")
+    @Size(min = 3, max = 50, message = "Breed must be between 3 and 50 characters")
+    private String breed;
+
+    @NotBlank(message = "Age is required")
+    private double age;
+
+    @NotBlank(message = "Weight is required")
+    private double weight;
+
+    @NotBlank(message = "Birthdate is required")
+    private Date birthdate;
+
+    @Size(max = 500, message = "Medications too long!")
```

Comment:
Is there a way for us to account for multiple medications per pet? IE, and arraylist or other collection?